src/main/java/net/greatstart/controllers/ProjectController.java

```diff
@@ -1,39 +1,88 @@
 package net.greatstart.controllers;
 
 import net.greatstart.model.Project;
+import net.greatstart.model.ProjectDescription;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.greatstart.model.ProjectDescription'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)